apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/PooledSession.cs

```diff
@@ -101,14 +105,14 @@ public sealed class PooledSession : IDisposable
 
         private PooledSession(SessionPool.ISessionPool pool, SessionName sessionName, ByteString transactionId, TransactionOptions transactionOptions, Timestamp readTimestamp, DateTime evictionTime, long refreshTicks)
```

Comment:
It's probably worth adding docs to this constructor, in terms of when it's used.
Because we can end up with transaction options but no transaction ID, as far as I can see (unless I've misunderstood) - it's just we presumably use a different constructor in that case?

Reply:
(Unless this happens later on, of course - or my mental model is wrong. Will check, and resolve this comment if it becomes obvious.)